src/applications/resources-and-support/components/SearchBar.jsx

```diff
@@ -29,7 +29,7 @@ export default function SearchBar({
   };
 
   return (
-    <div className="vads-u-border-top--2px vads-u-border-color--gray-light vads-u-padding-y--3 vads-u-padding-x--1">
+    <div className="medium-screen:vads-u-border-top--2px small-screen:vads-u-border-bottom--0px vads-u-border-color--gray-light vads-u-padding-y--3 vads-u-padding-x--1">
```

Comment:
So we typically have mobile as default styles, then desktop styles prefixed with `medium-screen:`, so for example if you wanted the border bottom to be 0 on mobile but 1px on desktop, you would do `vads-u-border-bottom--0px medium-screen:vads-u-border-bottom--1px` 🙂

Reply:
I almost never use `small-screen:` because of ^^

Reply:
ah yes, mobile first!